src/stylesheets/core/_notifications.scss

```diff
@@ -107,22 +50,91 @@ $uswds-notifications:
+  + "\A 2.10.0:"
+  + "\A - We updated the Spanish language link to USAGov in the identifier. "
+  + "\A   The link should now read 'Visite USAGov en Español'. "
+  + "\A - We improved the accessibility of SVG images in the banner. "
+  + "\A   See the full release notes for more information."
+  + "\A --------------------------------------------------------------------"
+  + "\A 2.10.1:"
+  + "\A - We updated the `$theme-breadcrumb-background-color` default"
+  + "\A   to `default` from `'white'`. "
+  + "\A - We updated the `$theme-alert-icon-size` default to `5` from `4`."
+  + "\A - [Footer] We updated `usa-footer__logo-heading` to use a `p`."
+  + "\A   instead of an `h3`. "
+  + "\A --------------------------------------------------------------------"
+  + "\A 2.11.0:"
+  + "\A - We updated some settings defaults:"
+  + "\A   $theme-alert-icon-size: 4 (was 5)"
+  + "\A   $theme-table-border-color: default (was 'ink')"
+  + "\A   $theme-table-header-text-color: default (was 'ink')"
+  + "\A   $theme-table-stripe-text-color: default (was 'ink')"
+  + "\A   $theme-table-text-color: default (was 'ink')"
+  + "\A --------------------------------------------------------------------"
+  + "\A 2.11.2:"
+  + "\A - We deprecated the `$theme-site-max-width` variable. We're using"
+  + "\A   `$theme-grid-container-max-width` instead."
+  + "\A - We replaced the `thumb_down_off_alt` icon with `thumb_down_alt`"
+  + "\A   in our default icon sprite."
+  + "\A --------------------------------------------------------------------"
+  + "\A 2.12.0:"
+  + "\A - We deprecated three settings:"
+  + "\A   $theme-input-tile-background-color-selected"
+  + "\A   $theme-input-tile-border-color"
+  + "\A   $theme-input-tile-border-color-selected"
+  + "\A   Each of these is now calculated automatically"
+  + "\A - We updated some settings defaults:"
+  + "\A   $theme-color-success-dark: green-cool-50v (was green-cool-50)"
+  + "\A   $theme-color-success-darker: green-cool-60v (was green-cool-80)"
+  + "\A --------------------------------------------------------------------"
+  + "\A 2.12.2:"
+  + "\A - As a security precaution, we're now sanitizing text passed into "
+  + "\A   Tooltip. Tooltip no longer supports markup in its content."
+  + "\A --------------------------------------------------------------------"
+  + "\A 2.13.0:"
+  + "\A   BREAKING CHANGES:"
+  + "\A - We made a couple breaking changes to support the accessibility"
+  + "\A   of icon-only buttons. See complete release notes:"
+  + "\A   https://github.com/uswds/uswds/releases/tag/v2.13.0"
+  + "\A - We updated markup in the small Search button to include the "
+  + "\A   magnifying glass as an inline image instead of a background image."
+  + "\A   See the release notes for the updated markup."
+  + "\A - We updated the social icons in the Footer to use inline images "
+  + "\A   instead of a background images."
+  + "\A   See the release notes for the updated markup.";
```

Comment:
These are the `2.13.0` notes. We really want to call out that there are a couple breaking changes and direct folks to the release notes for update markup.

The link to the release notes is not yet active, but will be after the release.